internal/services/notification/db/db.go

```diff
@@ -214,11 +226,8 @@ func (d *DB) GetLastRunEventSequence(tx *sql.Tx) (*types.LastRunEventSequence, e
 	return out, errors.WithStack(err)
 }
 
-func (d *DB) GetCommitStatusDeliveriesAfterSequence(tx *sql.Tx, afterSequence uint64, deliveryStatus types.DeliveryStatus, limit int) ([]*types.CommitStatusDelivery, error) {
+func (d *DB) GetProjectCommitStatusDeliveriesAfterSequence(tx *sql.Tx, afterSequence uint64, limit int) ([]*types.CommitStatusDelivery, error) {
```

Comment:
`GetProject...` is wrong since there's no filter by project. Probably also the function you copied has the same issue.

Reply:
the other function with the same issue is `GetProjectRunWebhookDeliveriesAfterSequence`, should I change in this PR with GetRunWebhookDeliveriesAfterSequence or I must open a new PR?

Reply:
Please open a new PR with a clear/detailed commit.

Reply:
opened PR #467 to fix